R/chrome.R

```diff
@@ -172,6 +172,15 @@ chrome_print = function(
     if (missing(output) && !file.exists(input))
       output = xfun::with_ext(basename(gsub('[#?].*', '', url)), format)
     output2 = normalizePath(output, mustWork = FALSE)
+    # try to remove the file and throw a clear error if it still exists as it may be locked
+    if (!suppressWarnings(file.remove(output2)) && xfun::file_exists(output2)) {
```

Comment:
In theory your code is more robust, but practically this may be enough: https://github.com/yihui/knitr/blob/8fa7d1710bae72f9636ef64fa9286669329f1a9c/R/utils-conversion.R#L118

```suggestion
    if (xfun::file_exists(output2) && !file.remove(output2)) {
```

Reply:
It is just that `file.remove` threw a warning for me, at least on Windows. Do we want to keep showing this warning ?

Reply:
Oh good point. Then please feel free to discard my suggestion.